clients/js/src/generated/instructions/mintToCollectionV1.ts

```diff
@@ -238,11 +238,9 @@ export function mintToCollectionV1(
     resolvedAccounts.compressionProgram.isWritable = false;
   }
   if (!resolvedAccounts.tokenMetadataProgram.value) {
-    resolvedAccounts.tokenMetadataProgram.value = context.programs.getPublicKey(
-      'mplTokenMetadata',
-      'metaqbxxUerdq28cj1RbAWkYQm3ybzjb6a8bt518x1s'
+    resolvedAccounts.tokenMetadataProgram.value = publicKey(
+      'BGUMAp9Gq7iTEuizy4pqaxsTyUCBK68MDfK752saRPUY'
     );
-    resolvedAccounts.tokenMetadataProgram.isWritable = false;
```

Comment:
Interesting do you know why `isWritable` was being set to false here prior to this change, when it seemingly didn't need to be done because it was already that way from in the code above?

Reply:
Honestly no idea. I wonder if Kinobi just defaults to very explicit setting of each account meta.